recipe/meta.yaml

```diff
@@ -64,12 +64,13 @@ test:
 
 about:
   home: https://github.com/ulmo-dev/ulmo
-  license: BSD 3-Clause
+  license: BSD-3-Clause
+  license_file: LICENSE
   summary: 'Clean, simple, and fast access to public hydrology and climatology data'
   doc_url: http://ulmo.readthedocs.io/
   dev_url: https://github.com/ulmo-dev/ulmo/
 
 extra:
   recipe-maintainers:
+    - emiliom
     - ocefpaf
-    - dharhas
```

Comment:
@dharhas do we have your :+1: here?

Reply:
FYI, I became the maintainer of ulmo ~ 18 months ago

Reply:
This is about the conda-forge feedstock. We cannot remove someone without their OK.

PS: congrats on the commit rights upstream!